core/bones/relational.py

```diff
@@ -141,6 +151,12 @@ def __init__(
             self.parentKeys = parentKeys
 
         self.using = using
+        if isinstance(updateLevel, int):
+            logging.warning(f"parameter updateLevel = {updateLevel} in RelationalBone is deprecated."
+                            f"Please use the RelationalUpdateLevel enum instead")
+            warnings.warn(
+                f"parameter updateLevel = {updateLevel} in RelationalBone is deprecated.", DeprecationWarning
+            )
```

Comment:
This cannot only be a warning!
```
if not skel[boneName] or self.updateLevel == RelationalUpdateLevel.OnValueAssignment:
```
would be `False` in case of `updateLevel = 2`

We have to convert it to a enum value or raise an exception.

Reply:
I would prefer converting this into the appropriate enum.